Source/WebCore/css/CSSCustomPropertyValue.h

```diff
@@ -107,6 +107,18 @@ class CSSCustomPropertyValue final : public CSSValue {
 
     Ref<const CSSVariableData> asVariableData() const;
 
+    // Say true conservatively.
+    bool customMayDependOnBaseURL() const { return true; }
```

Comment:
Did you actually see problems with allowing this? Custom properties are resolved and applied dynamically during style resolution so they shouldn't affect cacheability.

Reply:
One of the issue I found is that `CSSCustomPropertySyntax::Type::URL` can include URL which can be a BaseURL. And to detect this, we need to have some SyntaxValue traversing code too. So for now, I disabled this.